plugins/modules/lvg.py

```diff
@@ -72,6 +72,29 @@
     - If C(true), allows to remove volume group with logical volumes.
     type: bool
     default: false
+  old_vg:
+    description:
+    - The old name of the volume group.
+    - If specified, the volume group is renamed.
+    type: str
+    required: false
+  active:
+    description:
+    - Whether the volume group's logical volumes are active and visible to the host.
```

Comment:
I was wondering too. Then it felt it would require too many unrelated code changes. Now, it makes sense to implement the new states. It would clarify the defined task's intention. I'll refactor the module.